prime-router/src/main/kotlin/cli/tests/TestReportStream.kt

```diff
@@ -802,10 +802,10 @@ abstract class CoolTest {
 
         lateinit var allGoodReceivers: MutableList<Receiver>
         lateinit var allGoodCounties: String
-        const val historyTestOrgName = "historytest"
+        const val historytestOrgName = "historytest"
```

Comment:
Why do you change from the camelCase variable?  Notice, it is in sync with the historyTestSender variable below.